adapter/thing/car_charger_test.go

```diff
@@ -0,0 +1,589 @@
+package thing_test
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/futurehomeno/fimpgo"
+	"github.com/futurehomeno/fimpgo/fimptype"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/service/chargepoint"
+	"github.com/futurehomeno/cliffhanger/adapter/service/meterelec"
+	"github.com/futurehomeno/cliffhanger/adapter/thing"
+	"github.com/futurehomeno/cliffhanger/router"
+	"github.com/futurehomeno/cliffhanger/task"
+	mockedchargepoint "github.com/futurehomeno/cliffhanger/test/mocks/adapter/service/chargepoint"
+	mockedmeterelec "github.com/futurehomeno/cliffhanger/test/mocks/adapter/service/meterelec"
+	"github.com/futurehomeno/cliffhanger/test/suite"
+)
+
+var (
+	evChargerTestStates        = []string{"ready_to_charge", "charging", "error"}
+	evChargerChargingTestModes = []string{"normal", "slow"}
+
+	errTest = errors.New("oops")
+)
+
+func TestRouteCarCharger(t *testing.T) { //nolint:paralleltest
```

Comment:
You did not test out in routing scenarios for:
* "missing device" in adapter for each command 
* wrong format of input commands
While obviously this is not super important, it's easy to add, and we're struggling to increase cov here.

Reply:
OK, done.